spec/views/external_resources/edit.html.erb_spec.rb

```diff
@@ -27,13 +29,22 @@
 
       assert_select "input[name=?]", "external_resource[phone]"
 
-      assert_select "input[name=?]", "external_resource[description]"
+      assert_select "textarea[name=?]", "external_resource[description]"
 
-      assert_select "input[name=?]", "external_resource[display_on_website]"
+      # binding.pry # TODO - helppppppppp
```

Comment:
Having trouble figuring out how to `assert_select` the date part inputs. Would LOVE help on this!